### PR TITLE
Added info about necessary SELinux & firewalld setup

### PR DESCRIPTION
**Pull request by nhr**
@bparees or @deads2k please review.

### CODE REVIEW

File: examples/sample-app/README.md [hunk 1]
```diff
@@ -11,18 +11,52 @@ https://github.com/openshift/origin/blob/master/examples/sample-app/container-se
...
+### FirewallD Changes ###
+
+Similar to our work on SELinux policies, the OpenShift firewalld rules are also a work in progress. For now it is easiest to disable firewalld altogether:
+
+    $ sudo systemctl stop firewalld
+    $ sudo systemctl disable firewalld
+
+### Reboot and Run ###
+
+Reboot your system and you will be ready to play with this OpenShift sample application.
```
Comment (bparees) [inline examples/sample-app/README.md]: There's no need to reboot if you just run "setenforce 0"

**Comment by bparees** [comment]
It might be better to just reference my troubleshooting doc which covers this issue:
https://github.com/openshift/origin/blob/master/docs/debugging-openshift.md

File: examples/sample-app/README.md (hunk 1 above)
Comment (nhr) [inline examples/sample-app/README.md]: My system hung when I disabled these things. For the purposes of setting up first-time users for the highest likelihood of first-time success, I'd rather suggest the hammer approach.

**Comment by nhr** [comment]
I will update the PR to reference this troubleshooting info (which is great, btw), but again, for maximum first-user success I think it is probably better to spell out everything in one place.

File: examples/sample-app/README.md (hunk 1 above)
Comment (bparees) [inline examples/sample-app/README.md]: i'm worried we're going to lose a lot of users at "reboot your system".  I hate rebooting and i'd never do it just to try something out unless it was incredibly important to me.

i'm not aware of anyone else having had their system hang, so i think we might want to gather a little more data on why it failed for you before breaking out that hammer.

**Comment by nhr** [comment]
Okay; removed reboot instruction and added a section to reference the troubleshooting guide.

File: examples/sample-app/README.md [hunk 2]
```diff
@@ -11,18 +11,52 @@ https://github.com/openshift/origin/blob/master/examples/sample-app/container-se
...
+    $ sudo vi /etc/sysconfig/selinux
+
+And change:
+
+    SELINUX=enforcing
+
+To:
+
+    SELINUX=permissive
+
```
Comment (bparees) [inline examples/sample-app/README.md]: does this mechanism take effect immediately like setenforce 0?

Comment (nhr) [inline examples/sample-app/README.md]: Ah, probably not. Thank you.

**Comment by nhr** [comment]
Commands are fixed for non-permanent change.

**Comment by bparees** [comment]
lgtm. [merge]

**Comment by smarterclayton** [comment]
[merge]

**Pull request merged**